### PR TITLE
Add a nonlocal example of follower stopper as requested

### PR DESCRIPTION
**Pull request by eugenevinitsky**

### CODE REVIEW

File: flow/controllers/velocity_controllers.py [hunk 1]
```diff
@@ -116,6 +116,53 @@ def get_accel(self, env):
...
 
 
+class NonLocalFollowerStopper(FollowerStopper):
+    """Follower stopper that uses the average system speed to compute its acceleration."""
+
+    def get_accel(self, env):
+        """See parent class."""
+        lead_id = env.k.vehicle.get_leader(self.veh_id)
+        this_vel = env.k.vehicle.get_speed(self.veh_id)
+        lead_vel = np.mean(env.k.vehicle.get_speed(env.k.vehicle.get_ids()))
```
Comment (AboudyKreidieh) [inline flow/controllers/velocity_controllers.py]: My understanding was they wanted us to do something like

```
self.v_des = np.mean(env.k.vehicle.get_speed(env.k.vehicle.get_ids()))
```

Are you sure this is what they wanted?

Comment (eugenevinitsky) [reply flow/controllers/velocity_controllers.py]: Honestly not clear on what they wanted but what you have above makes more sense.

Comment (Yasharzf) [reply flow/controllers/velocity_controllers.py]: @AboudyKreidieh is right. They wanted to assign the speed to the average speed.

Comment (AboudyKreidieh) [reply flow/controllers/velocity_controllers.py]: k, if we all agree I can modify and merge it

Comment (eugenevinitsky) [reply flow/controllers/velocity_controllers.py]: Fixed.

**Pull request merged**